sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -79,7 +90,7 @@ object AkkaserverlessPlugin extends AutoPlugin {
         akkaGrpcCodeGeneratorSettings.value :+ AkkaserverlessGenerator.enableDebug) -> (Test / sourceManaged).value,
     Compile / generateUnmanaged := {
       Files.createDirectories(Paths.get((Compile / temporaryUnmanagedDirectory).value.toURI))
-      // Make sure generation has happened
+      // Make sure generation has happened and that the protobuf descriptor file is created
```

Comment:
That doesn't work as intended, because there's no guarantee that the descriptor has been generated when `managedResources` are copied to the classpath.

Reply:
Will fix this.